README.md

```diff
@@ -26,12 +29,14 @@ This package provides OpenTelemetry-compliant tracing to .NET applications for t
 To install Cisco OpenTelemetry Distribution simply run:
 
 ```sh
-dotnet add package Cisco.Otel.Distribution
+dotnet add package Cisco.Telescope
 ```
 
 ### Library initialization
 
-> Cisco OpenTelemetry Distribution is activated and instruments the supported libraries once the module is imported.
+Cisco OpenTelemetry Distribution is activated and instruments the supported libraries once the `ciscoTracing.init()` has returned.
```

Comment:
In web apps, we add it as a middleware, which is run when the application is started.
Maybe we can write something like "Cisco OpenTelemetry Distribution is activated and instruments the supported libraries once the Cisco Tracer has been initialized"